packages/core/src/browser/tree/search-box.ts

```diff
@@ -171,8 +172,9 @@ export class SearchBox extends BaseWidget {
         this.addClass(SearchBox.Styles.SEARCH_BOX);
 
         const input = document.createElement('input');
-        input.readOnly = true;
+        input.readOnly = false;
         input.type = 'text';
+        input.onkeydown = (ev) => this.handle.bind(this)(ev);
```

Comment:
You can write this as:

```ts
input.onkeydown = ev => this.handle.bind(this)(ev);
```

It is the reason for the build failing https://travis-ci.org/theia-ide/theia/jobs/549920261#L861